src/otx/engine/engine.py

```diff
@@ -323,9 +323,41 @@ def predict(
             return_predictions=return_predictions,
         )
 
-    def export(self, *args, **kwargs) -> None:
-        """Export the trained model to OpenVINO Intermediate Representation (IR) or ONNX formats."""
-        raise NotImplementedError
+    def export(self, checkpoint: str | Path | None = None, export_config: ExportConfig | None = None, **kwargs) -> Path:
```

Comment:
```suggestion
    def export(self, checkpoint: str | Path | None = None, export_config: ExportConfig | None = None) -> Path:
```
How about removing `kwargs` in argument to reduce user confusion?